src/edu/csus/ecs/pc2/exports/ccs/ResultsFile.java

```diff
@@ -325,7 +266,7 @@ public String[] createTSVFileLines(IInternalContest contest, Group group, String
      * Determine and return award medal color.
      *
      * <pre>
-     * Award is a string with value "gold", "silver", "bronze", "ranked"
+     * Award is a string with value "gold", "silver", "bronze", "highest honor", "high honor", "honor"
```

Comment:
I think this new "definition" of method `getAwardMedal()` could be problematic.  Previously, the four possible return values (`gold`, `silver`, `bronze`, and `ranked`) were mutually exclusive.  Now, a team could be both `gold` AND `highest honor` (for example).   

It seems like there should be one method `getAwardMedal()` which operates (somewhat) like the old method (i.e., it returns what really is a "medal status") and a _second, new_ method like `getHonorsCategory()` which returns the Honors category without confusing/overlapping it with "medal color".   Otherwise, we're likely to have trouble down the road when developers call `getAwardMedal()` and expect that what they'll get back is truly a medal designator (which it might not be under this new method definition).

Reply:
This is not a new "definition", since previously, "**ranked**" and "**honorable**" were returned, and, well, they are not medals.  I am thinking the name of the method should be simply changed to "`getAwardCitation()`" since, that is what it actually is.  The definition of the field in the `results.tsv` is wishy-washy at best, and wrong at worst: "**medalCitation**".  This is only used by the generation of the `results.tsv` and `results.csv` files to populate that specific field (**medalCitation**).  My vote is to simply change its name to "`getAwardCitation()`" which is what it really is supposed to be.

Reply:
Updated